packages/vuetify/src/composables/__tests__/dimensions.spec.ts

```diff
@@ -0,0 +1,147 @@
+import { reactive } from 'vue'
+
+import {
+  dimensionsFactory,
+} from '../dimensions'
+
+describe('dimensions.ts', () => {
```

Comment:
Don't really need the top group, jest groups everything by file anyway
![image](https://user-images.githubusercontent.com/16421948/97068666-6637f680-1615-11eb-9018-fba3485dba7f.png)